src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -145,6 +151,7 @@ namespace ProfileEvents
 namespace CurrentMetrics
 {
     extern const Metric DelayedInserts;
+    extern const Metric AttachedDataParts;
```

Comment:
Unify this block with https://github.com/ClickHouse/ClickHouse/pull/57091/files#diff-b7d33a7abda5f3365d5e4fb58ca98b756f6401372b2e2d5eb449a2bd3444fb5eR127